O21.Game/Localization/Translations.fs

```diff
@@ -0,0 +1,41 @@
+module O21.Localization.Translations
+
+open FSharp.Data
+open System.IO
+open LocalizationPaths
+
+type TranslationLanguageType = 
+    | Json
+
+type HelpRequestType =
+    | MarkdownHelp
+    | RussianHelp
+                    
+type Language = { 
+    Name: string
+    Type: TranslationLanguageType
+    HelpRequestType: HelpRequestType
+}
+
+type private Provider = JsonProvider<"Localization/Translations/english.json">
+
+let Translation(language: Language) =
+    Provider.Load (match language.Type with
+                    | Json -> $"{TranslationsFolder()}{language.Name.ToLowerInvariant()}.json")
+
+let AvailableLanguages = 
+    let files = Directory.GetFiles(TranslationsFolder(), "*.json")
+    seq {
+        for file in files do
+            let fileName = (Path.GetFileNameWithoutExtension file).ToLowerInvariant()
```

Comment:
`ToLowerInvariant()` is unnecessary and could break on a case-sensitive FS, let's avoid that.